src/features/workflow/Workflow.tsx

```diff
@@ -34,7 +34,7 @@ const Workflow: React.FC<WorkflowProps> = ({ qriRef }) => {
   const location = useLocation<WorkflowLocationState>()
   const workflow = useSelector(selectWorkflow)
   const workflowDataset = useSelector(selectWorkflowDataset)
-  const latestRun = useSelector(selectLatestRun)
+  const latestRun = useSelector(selectLatestDryRun)
```

Comment:
could this be the source of the bug where only the dry run was showing updating the outline?

Reply:
I'm having trouble reproducing hat error, so it may be a fluke.